rust/processor/src/db/common/models/token_v2_models/parquet_v2_collections.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+// This is required because a diesel macro makes clippy sad
+#![allow(clippy::extra_unused_lifetimes)]
+#![allow(clippy::unused_unit)]
+
+use field_count::FieldCount;
+use serde::{Deserialize, Serialize};
+
+// PK of current_collections_v2, i.e. collection_id
+pub type CurrentCollectionV2PK = String;
+
+#[derive(Clone, Debug, Deserialize, FieldCount, Serialize)]
+pub struct CollectionV2 {
+    pub txn_version: i64,
+    pub write_set_change_index: i64,
+    pub collection_id: String,
+    pub creator_address: String,
+    pub collection_name: String,
+    pub description: String,
+    pub uri: String,
+    pub current_supply: u64,
+    pub max_supply: Option<u64>,
+    pub total_minted_v2: Option<u64>,
+    pub mutable_description: Option<bool>,
+    pub mutable_uri: Option<bool>,
+    pub table_handle_v1: Option<String>,
+    pub token_standard: String,
+    pub transaction_timestamp: chrono::NaiveDateTime,
```

Comment:
`transaction_timestamp` (typo) -> `block_timestamp`

same questions as above about supply

this includes v1 right?

Reply:
this includes v1.